bundles/com.zeligsoft.domain.dds4ccm/META-INF/MANIFEST.MF

```diff
@@ -32,7 +32,8 @@ Require-Bundle: org.eclipse.ui;bundle-version="3.113.0",
  org.eclipse.papyrus.infra.services.validation;bundle-version="3.0.0",
  org.eclipse.emf.compare;bundle-version="3.5.3",
  org.eclipse.emf.compare.rcp;bundle-version="2.5.2",
- org.eclipse.emf.common
+ org.eclipse.emf.common,
+ org.eclipse.papyrus.infra.emf.gmf;bundle-version="1.2.100"
```

Comment:
Remove dependency to GMF

Reply:
I've pushed a commit with these changes. It works with RecordingCommand. Can you review it? Thanks.